common/scala/src/main/scala/org/apache/openwhisk/http/ErrorResponse.scala

```diff
@@ -72,7 +72,7 @@ object Messages {
 
   /** Standard message for too many concurrent activation requests within a time window. */
   def tooManyConcurrentRequests(count: Int, allowed: Int) =
-    s"Too many concurrent requests in flight (count: $count, allowed: $allowed)."
+    s"Too many concurrent requests in flight (currently running: $count, allowed per controller: $allowed)."
```

Comment:
```suggestion
    s"Too many concurrent requests (count in window: $count, allowed per controller: $allowed)."
```

Reply:
Suggest a change to address @bdoyle0182 's comment

Reply:
@dgrove-oss @bdoyle0182 @mrutkows suggested a change in a separate PR linked below to keep it clean. I'll just close the other PR if I'm on the wrong track ;)

Reply:
@rabbah - do you have any thoughts?  This is a part of the controller I don't know very well.

Reply:
LGTM.